PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -591,23 +702,33 @@ DECLARE_SOA_DYNAMIC_COLUMN(InvMassK0s, invMassK0s, //! mass under K0short hypoth
 
 namespace hf_reso_track
 {
-DECLARE_SOA_COLUMN(Px, px, float);                   //! x-component of momentum
-DECLARE_SOA_COLUMN(Py, py, float);                   //! y-component of momentum
-DECLARE_SOA_COLUMN(Pz, pz, float);                   //! z-component of momentum
-DECLARE_SOA_COLUMN(Sign, sign, uint8_t);             //! charge sign
-DECLARE_SOA_COLUMN(NSigmaTpcPi, nSigmaTpcPi, float); //! TPC Nsigma for pion hypothesis
-DECLARE_SOA_COLUMN(NSigmaTpcKa, nSigmaTpcKa, float); //! TPC Nsigma for kaon hypothesis
-DECLARE_SOA_COLUMN(NSigmaTpcPr, nSigmaTpcPr, float); //! TPC Nsigma for proton hypothesis
-DECLARE_SOA_COLUMN(NSigmaTofPi, nSigmaTofPi, float); //! TOF Nsigma for pion hypothesis
-DECLARE_SOA_COLUMN(NSigmaTofKa, nSigmaTofKa, float); //! TOF Nsigma for kaon hypothesis
-DECLARE_SOA_COLUMN(NSigmaTofPr, nSigmaTofPr, float); //! TOF Nsigma for proton hypothesis
-DECLARE_SOA_COLUMN(HasTof, hasTof, bool);            //! flag for presence of TOF
-DECLARE_SOA_DYNAMIC_COLUMN(Pt, pt,                   //!
+DECLARE_SOA_COLUMN(Px, px, float);                             //! x-component of momentum
+DECLARE_SOA_COLUMN(Py, py, float);                             //! y-component of momentum
+DECLARE_SOA_COLUMN(Pz, pz, float);                             //! z-component of momentum
+DECLARE_SOA_COLUMN(Sign, sign, uint8_t);                       //! charge sign
+DECLARE_SOA_COLUMN(NSigmaTpcPi, nSigmaTpcPi, float);           //! TPC Nsigma for pion hypothesis
+DECLARE_SOA_COLUMN(NSigmaTpcKa, nSigmaTpcKa, float);           //! TPC Nsigma for kaon hypothesis
+DECLARE_SOA_COLUMN(NSigmaTpcPr, nSigmaTpcPr, float);           //! TPC Nsigma for proton hypothesis
+DECLARE_SOA_COLUMN(NSigmaTofPi, nSigmaTofPi, float);           //! TOF Nsigma for pion hypothesis
+DECLARE_SOA_COLUMN(NSigmaTofKa, nSigmaTofKa, float);           //! TOF Nsigma for kaon hypothesis
+DECLARE_SOA_COLUMN(NSigmaTofPr, nSigmaTofPr, float);           //! TOF Nsigma for proton hypothesis
+DECLARE_SOA_COLUMN(HasTof, hasTof, bool);                      //! flag for presence of TOF
+DECLARE_SOA_COLUMN(NumItsCls, numItsCls, int);                 //! Number of clusters in ITS
+DECLARE_SOA_COLUMN(NumTpcCrossedRows, numTpcCrossedRows, int); //! Number of TPC crossed rows
+DECLARE_SOA_COLUMN(Chi2Tpc, chi2Tpc, float);                   //! TPC chi2
```

Comment:
There is a lot of duplicated column declarations throughout the header. Please declare them only once.